ethvm/main.cpp

```diff
@@ -226,7 +246,7 @@ int main(int argc, char** argv)
 	};
 
 	executive.initialize(t);
-	if (contractCodeString != "")
+	if (!contractCodeString.empty())
```

Comment:
That makes absolutely no sense.

- `--code A --input B` will create a contract with init code A, input B is ignored,
- `--input B` will call empty code with input B.
- `--code A` will create a contract with init code A.